### DISCUSSION TITLE
Can't find type export `IconValue`

### DISCUSSION THREAD

**Discussion by frederikheld** [vuetifyjs/vuetify#20606] [Q&A]
Hey everyone,

I'd like to use the type `IconValue` but I can't find a type export. Internally, Vuetify uses `import { IconValue } from '@/composables/icons'` but this is obviously not the way to import it from the build.

Does anyone have an idea where to find it? From all paths `vuetify/*` I have tried so far I get the message "Module '"xx"' declares 'IconValue' locally, but it is not exported." Is this type even publicly available?

**Comment by Firedan1176** [comment]
I am running into this same scenario but for `SnackbarMessage`, which was created for the `VSnackbarQueue` labs feature. My only guess is that the vuetify build process strips away exported types somehow and for some reason.

My related discussion is [here](https://github.com/vuetifyjs/vuetify/discussions/20952)

**Edit**: This is discussed as an issue **#20484**, so it appears it's being looked at.
